app/assess/templates/resolve_flag.html

```diff
@@ -13,6 +13,32 @@
 </p>
   {{ logout_partial(sso_logout_url) }}
 </div>
+
+{% if form.errors.get(form.resolution_flag.id) or form.errors.get("justification") %}
+<div class="govuk-error-summary" data-module="govuk-error-summary">
+  <div role="alert">
+    <h2 class="govuk-error-summary__title">
+      There is a problem
+    </h2>
+    <div class="govuk-error-summary__body">
+      <ul class="govuk-list govuk-error-summary__list">
+        {% if form.errors.get(form.resolution_flag.id) %}
+        <li>
+          <a href=#{{form.resolution_flag.id}}>This field is required.</a>
+        </li>
+        {% endif %}
+        {% if form.errors.get("justification") %}
+        <li>
+          <a href=#{{form.justification.id}}>This field is required.</a>
```

Comment:
In your last screenshot, these two errors are duplicated - at the top of the screen, they don't really give the user much context of the section where they happened, should we change these errors to be more descriptive, or denote which section they refer to?